src/pykeen/models/nbase.py

```diff
@@ -651,10 +657,13 @@ def _get_representations(
         h: Optional[torch.LongTensor],
         r: Optional[torch.LongTensor],
         t: Optional[torch.LongTensor],
+        invert_relation: bool = False,  # TODO: do we need this here?
```

Comment:
TODO: do we need this?

Reply:
I think this would be necessary if we had a separate Representation module for inverse relations 🤔